### ISSUE TITLE
chdir option?

### ISSUE THREAD

**Issue by t-kalinowski** [rticulate/import#15]
Hello,

Thanks for this package, lately it's become an integral part of my workflow. 

Usually, I use `import` to import some specific objects from R scripts that otherwise would pollute my global environment. One issue I've run into is that if I do `import::here(my_obj, .from = 'path/to/script.R')`, the script is actually sourced in working directory `path/to/`. Would it be possible to give an option to disable this behavior? I'm currently organizing all the scripts within a project to assume the same working directory, but organize the scripts in nested folders as appropriate. Having an ability to set `chdir = FALSE` would be very usefully.  Ideally the default could be something like `getOption("import.chdir", TRUE)`.

**Comment by smbache** [comment]
Sounds like a reasonable request!

**Comment by torfason** [comment]
This has now been fixed on master. If you could install the most current version and test whether the fix works for you (and if the new version works for you in general) that would be greatly appreciated. You can get the fix with:
`remotes::install_github("smbache/import")`

The relevant new parameter is `.chdir` (although you may also be interested in the `.directory` parameter, given your use case. Here is a usage example (based the unit tests):

``` r
import::from(module_chdir.R, fun_chdir_report, .directory="module_chdir", .chdir=TRUE)
fun_chdir_report()
#> [1] "Module was sourced in: .../import/tests/test_import/module_chdir"
```

``` r
import::from(module_chdir.R, fun_chdir_report, .directory="module_chdir", .chdir=FALSE)
fun_chdir_report()
#> [1] "Module was sourced in: .../import/tests/test_import"
```

**Comment by torfason** [comment]
Fixed in release 1.2.0
